Common/Securities/Cash.cs

```diff
@@ -70,13 +70,19 @@ public class Cash
         /// </summary>
         public decimal ValueInAccountCurrency => Amount * ConversionRate;
 
+        /// <summary>
+        /// Base account currency
+        /// </summary>
+        public string AccountCurrency { get; internal set; }
```

Comment:
Instead of having an account currency property, it may be better to accept the account currency as an argument to the only method that requires the data.